kotest-assertions/kotest-assertions-shared/src/commonMain/kotlin/io/kotest/matchers/should.kt

```diff
@@ -47,9 +47,9 @@ fun <T> invokeMatcher(t: T, matcher: Matcher<T>): T {
       when (result) {
          is ComparableMatcherResult -> errorCollector.collectOrThrow(
             failure(
-               Expected(Printed(result.expected())),
-               Actual(Printed(result.actual())),
-               result.failureMessage()
+               expected = Expected(Printed(result.expected())),
+               actual = Actual(Printed(result.actual())),
+               prependMessage = result.failureMessage() + "\n"
```

Comment:
This newline breaks some tests. I'm pushing a commit that updates the tests, but feel free to drop it and resolve in another way.